inputs/prod_inner.json

```diff
@@ -48,6 +48,10 @@
       "name": "add_dockerfile"
     },
     {
+      "name": "add_help",
+      "required": false
```

Comment:
This is listed after `add_dockerfile` and I initially thought it would add in the Dockerfile as it was prior to the help ADD instruction being added, but that plugin instructs the Dockerfile to store itself inside the built image and does not take a copy so that seems fine.